### ISSUE TITLE
Stage 1.4: Integration - Freeze root embeddings for downstream models

### ISSUE THREAD

**Issue by marctjones** [marctjones/klareco#620]
# Task

Freeze root embeddings for use in downstream models (M1, Entity Classifier, Reranker).

## Parent Epic

Part of **#616** (EPIC: Tier-Filtered Model Retraining)

### 1. Update CompositionalEmbedding to load new roots
```python
# klareco/embeddings/compositional.py
checkpoint = torch.load('models/root_embeddings/best_model.pt')
self.root_embed.weight.data = checkpoint['model_state_dict']['embeddings.weight']
self.root_vocab = checkpoint['root_to_idx']
```

### 2. Freeze root embeddings (no gradient updates)
```python
self.root_embed.weight.requires_grad = False
```

### 3. Verify downstream models can load
- M1 selectional: `klareco/models/m1_selectional.py`
- Entity classifier: `klareco/models/entity_classifier.py`
- Reranker: `klareco/rag/reranker.py`

### 4. Update training scripts
All downstream training scripts should use frozen root embeddings:
- `scripts/train_m1_selectional.py`
- `scripts/train_entity_classifier.py`
- `scripts/train_reranker.py`

## Acceptance Criteria

- [ ] CompositionalEmbedding loads new root embeddings
- [ ] Root embeddings frozen (requires_grad=False)
- [ ] Vocabulary size verified (~9,800 roots)
- [ ] No function words in loaded vocabulary
- [ ] All downstream models can initialize without errors
- [ ] Training scripts updated to use frozen embeddings

## Validation

```python
# Test that embeddings are frozen
from klareco.embeddings.compositional import CompositionalEmbedding
comp_emb = CompositionalEmbedding(...)
assert comp_emb.root_embed.weight.requires_grad == False
print(f"Vocab size: {len(comp_emb.root_vocab)}")  # ~9,800
```

## Dependencies

- Blocked by: #619 (Test quality)
- Blocks: #621 (M1 data generation), #625 (Entity classifier data), #629 (Reranker data)

**Comment by marctjones** [comment]
Closing as part of a 2026-09-08 backlog cleanup (see PLAN.md, DESIGN.md, CLAUDE.md's merge gate).

This issue predates the merge gate (adopted 2026-07-13) and the AST-native orchestrator rewrite. It describes work against the pre-orchestrator RAG pipeline or the learned model stack (M1/M2 embeddings, compositional embeddings, entity classifiers, neural rerankers) that was pruned from the repo -- klareco/embeddings/, klareco/models/, and klareco/summarization/ no longer exist. It has no milestone and was never benchmarked, matching exactly the failure mode CLAUDE.md documents (25 capability issues filed 2026-05-26, 8 implemented, none benchmarked).

Labeled archived-2026-09 so this batch is recoverable: gh issue list --state closed --label archived-2026-09. If this capability is still wanted, re-file it against the current architecture naming the benchmark metric and test the merge gate requires.
